contents/handbook/strategy/overview.md

```diff
@@ -58,38 +58,32 @@ While there is value in the items on the right, we value the items on the left m
 
 * _Breadth_: build basic versions of every feature needed rather than a small set of sophisticated ones, so our customers can consolidate.
 
-**High quality users vs. high paying businesses**
+**MidMarket vs. Enterprise**
 
-* _High quality users_: focus on acquiring more high quality users over big ticket contracts, so we can get better feedback and learn faster.
+* _MidMarket_: focus on acquiring more high quality MidMarket customers (who may start using for free) over big ticket contracts, so we can get better feedback and learn faster.
```

Comment:
`Mid-market` or `mid market` maybe?

Reply:
Yes, this is triggering for me! 😂 `Mid-market` would be best.